cmd/edit/kubeletconfig/cmd.go

```diff
@@ -97,19 +97,15 @@ func run(_ *cobra.Command, _ []string) {
 		r.Reporter.Infof("Enabling interactive mode")
 	}
 
+	maxPidsLimit, err := GetMaxPidsLimit(r.OCMClient)
+	if err != nil {
+		r.Reporter.Errorf("Failed to check maximum allowed Pids limit for for cluster '%s'",
```

Comment:
NP: `Pid limit`

Reply:
Or `Limit of Pids` would work :smile:

Reply:
This is obviously a nitpick, but my reasoning is that `allowed pids` is plural, but `allowed pid limit` should be singular. Much like your `mile limit` before your next oil change is not said `miles limit` -- or `car limit` in a parking garage vs `cars limit`

Reply:
@hunterkepley See my previous comment about the upstream field already being named `podPidsLimit`